src/Talis/Persona/Client/Login.php

```diff
@@ -206,7 +206,7 @@ protected function login($redirectUri = '', $query = null)
         $_SESSION[self::LOGIN_PREFIX . ':loginState'] = $loginState;
 
         // Log user in
-        $redirect = $this->config['persona_host'] . '/auth/providers/' . $_SESSION[self::LOGIN_PREFIX . ':loginProvider'] . '/login';
+        $redirect = $this->getPersonaHost() . '/auth/providers/' . $_SESSION[self::LOGIN_PREFIX . ':loginProvider'] . '/login';
```

Comment:
Distinct improvement.

Do we have access to any standardised URL-building facilities that ensure correct URL-encoding?